### PR TITLE
Increased fallible page coverage

### PR DESCRIPTION
**Pull request by Mac-Ste**
Fallible page now covers most pages that do not use complicated 'and/or' conditions and clickable/interactable elements.

### CODE REVIEW

File: src/test/kotlin/JiraPerformanceComparisonIT.kt [hunk 1]
```diff
@@ -38,7 +38,7 @@ class JiraPerformanceComparisonIT {
     fun shouldComparePerformance() {
         val results: List<EdibleResult> = AbruptExecutorService(newCachedThreadPool()).use { pool ->
             listOf(
-                benchmark("a.properties", JiraDcScenario::class.java, quality, pool),
+                benchmark("a.properties", JiraCloudScenario::class.java, quality, pool),
```
Comment (dagguh) [inline src/test/kotlin/JiraPerformanceComparisonIT.kt]: This shouldn't be needed for the `FalliblePageCoverage`

- [x] undo scenario change

File: custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt [hunk 2]
```diff
@@ -5,20 +5,22 @@ import org.openqa.selenium.By
 import org.openqa.selenium.WebDriver
 import org.openqa.selenium.support.ui.ExpectedConditions.or
 import org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated
+import kotlin.math.exp
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt]: - [x] Accidental import

File: custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt [hunk 3]
```diff
@@ -12,14 +13,20 @@ class CloudFiltersPage(
...
 
     override fun open(): FiltersPage {
         jira.navigateTo("secure/ManageFilters.jspa?sortKey=popularity&sortOrder=DESC&filterView=search")
         return this
     }
 
     override fun waitForList(): FiltersList {
-        val element = driver.wait(visibilityOfElementLocated(table))
-        return CloudFiltersList(element)
+        falliblePage.waitForPageToLoad()
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt:27]: So we even get more error-detection coverage thanks to the ease of reuse of the `FalliblePage` pattern. Nice!

File: custom-vu/src/main/kotlin/jces1209/vu/page/FalliblePage.kt [hunk 4]
```diff
@@ -56,6 +56,14 @@ class FalliblePage private constructor(
             )
         )
 
+        fun serverErrors() = potentialErrors(
```
Comment (dagguh) [inline custom-vu/src/main/kotlin/jces1209/vu/page/FalliblePage.kt:59]: I like how nicely it fits with the existing structure and semantics

**Review by dagguh — CHANGES_REQUESTED** [review]

File: custom-vu/src/main/kotlin/jces1209/vu/page/FalliblePage.kt (hunk 4 above)
Comment (Mac-Ste) [reply custom-vu/src/main/kotlin/jces1209/vu/page/FalliblePage.kt:59]: Yeah 😎

File: custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt (hunk 3 above)
Comment (Mac-Ste) [reply custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt:27]: Exactly, food for thought: maybe cover not only visibility but, clickable etc?

File: src/test/kotlin/JiraPerformanceComparisonIT.kt (hunk 1 above)
Comment (Mac-Ste) [reply src/test/kotlin/JiraPerformanceComparisonIT.kt]: Good catch, I run on cloud to ensure that it did not break anything

File: custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt (hunk 2 above)
Comment (Mac-Ste) [reply custom-vu/src/main/kotlin/jces1209/vu/page/CloudIssuePage.kt]: 😞

File: custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt (hunk 3 above)
Comment (dagguh) [reply custom-vu/src/main/kotlin/jces1209/vu/page/filters/CloudFiltersPage.kt:27]: Currently it's not very consistent. We'll be aligning with RAIL more (with focus on TTR and TTI), we'll address it then. Most likely we'll need both, so we can get `FalliblePage.waitForRender` and `FalliblePage.waitForInteractivity`.

**Review by dagguh — APPROVED** [review]

**Pull request merged**